src/include/OpenImageIO/imageio.h

```diff
@@ -2854,6 +2854,11 @@ OIIO_API std::string geterror(bool clear = true);
 ///    Colon-separated (or semicolon-separated) list of directories to search
 ///    for dynamically-loaded format plugins.
 ///
+/// - `string plugin_search_system_paths`
```

Comment:
It is an integer, would be nice to correct the comment if we are to stick to this implementation.